### PR TITLE
Launchpad

### PR DESCRIPTION
**Pull request by That-Canadian**
Initial Launchpad files

### CODE REVIEW

**Comment by drashna** [comment]
Could you add a readme for the board?  And if you're comfortable doing so, an `info.json` file for the board? 

https://docs.qmk.fm/reference/documentation-templates#keyboard-readme.md-template

https://docs.qmk.fm/hardware/keyboard-guidelines#info.json-format

File: keyboards/launchpad/rev1/rev1.h [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+#define REV1_H
+
+#include "../launchpad.h"
+
+//void promicro_bootloader_jmp(bool program);
+#include "quantum.h"
+
+//void promicro_bootloader_jmp(bool program);
+
+#define KEYMAP( \
```
Comment (drashna) [inline keyboards/launchpad/rev1/rev1.h]: We're deprecating KEYMAP in favor of LAYOUT. Would you change all instances of KEYMAP to LAYOUT?

**Comment by That-Canadian** [comment]
Done, replaced KEYMAP with LAYOUT, also added a README
@drashna

**Comment by drashna** [comment]
Awesome, thanks!

**Pull request merged**